client/js/widgets/button.js

```diff
@@ -247,6 +247,8 @@ export class Button extends Widget {
       }
 
       if(a.func == 'FLIP') {
+        if(a.limit !== undefined)
+          a.count = a.limit;
         setDefaults(a, { count: 0, face: null });
```

Comment:
There are likely going to be additional aliased parameters per function besides count as "limit".

Consider generalizing application of aliases, perhaps combined in a single function call with `setDefaults()` because definition of aliased parameters & parameter defaults should be considered together.

```js
if(a.fun == 'FLIP') {
  initParams(a, { count: "limit" },
    { count: 0, face: null },
  );
...
function initParams(array, aliases, defaults) {
  aliasParams(array, aliases);
  setDefaults(array, defaults);
}

function aliasParams(array, aliases) {
  aliases.forEach((alias, param) => {
    if(array[alias] !== undefined)
      array[param] = array[alias];
  });
}
```

Reply:
Since there seems to be ongoing discussion about whether and how to introduce aliases, but I think everyone agrees that the `in` relation and `type` qualifier in `SELECT` should proceed, perhaps we should separate these two things into two different PRs.